livingdocs-openapi.json

```diff
@@ -1341,6 +1349,101 @@
         }
       }
     },
+    "/api/v1/documents/{documentId}/commands": {
+      "patch": {
+        "security": [
+          {
+            "bearerAuth": [
+              "public-api:write"
+            ]
+          }
+        ],
+        "tags": [
+          "Command API"
+        ],
+        "operationId": "executeDocumentCommands",
+        "summary": "Execute Commands on Documents. Added in release-2023-11.",
+        "description": "Execute a Document Command based on its `id`. All commands run in a single transaction.",
+        "parameters": [
+          {
+            "name": "documentId",
+            "in": "path",
+            "description": "Id of the document",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "example": {
+                  "version": 1,
+                  "preconditions": [
+                    {
+                      "type": "isPublished",
+                      "value": true
+                    }
+                  ],
+                  "commands": [
+                    {
+                      "operation": "setMetadataProperty",
+                      "propertyName": "title",
+                      "value": "updated title",
+                      "oldValue": "previous title"
+                    },
+                    {
+                      "operation": "setEditableDirective",
+                      "componentId": "doc-1a2b3c4d5",
+                      "directiveName": "headline",
+                      "value": "updated headline"
+                    },
+                    {
+                      "operation": "publish"
+                    }
+                  ]
+                },
+                "properties": {
+                  "version": {
+                    "type": "number"
+                  },
+                  "preconditions": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "type": {
+                          "type": "string"
+                        }
+                      }
+                    }
+                  },
+                  "commands": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "operation": {
+                          "type": "string"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
```

Comment:
Could you added the other responses as well?

Reply:
It can have different types of error responses, but all with code 400. Then I would need to define some schemas. But I'm not sure i that's worth for the errors since they have all the same simple strcuture as we usually have with general errors, just the message is different depending on the operation.